cnxepub/formatters.py

```diff
@@ -775,9 +775,13 @@ def _replace_exercises(elem):
 def html_listify(tree, root_xl_element, extensions, list_type='ol'):
```

Comment:
Any chance you could throw a quick docstring up here? I seem to have lost track of what `extensions` is...